index.html

```diff
@@ -11,6 +13,90 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <div class="parent-container">
+    <header class="navcontainer">
```

Comment:
you may extract the whole header from the parent container so you may apply padding on the rest of the design, in line with the media width.